Packs/GitHub/ReleaseNotes/1_1_2.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### New: GitHub IAM
+- Integrate with GitHub's services to perform Identity Lifecycle Management operations.
+- Upgraded the Docker image to demisto/python3:3.8.6.13358.
```

Comment:
Do we need to show that we updated the Docker image if it's a brand new integration? If we can (and should) remove this, remove the "-" in front of the sentence beginning with "Integrate with..."

Reply:
Sounds good.